flake.nix

```diff
@@ -0,0 +1,107 @@
+{
+  description = "jujitsu";
```

Comment:
This project is called `jujutsu`, please update across this file. (`jujitsu` is also a common spelling, but not the right one here.)